packages/core/package.json

```diff
@@ -4,7 +4,7 @@
   "publishConfig": {
     "access": "public"
   },
-  "version": "1.5.0",
+  "version": "2.0.0",
```

Comment:
We bumped all packages to the same version for better consistency. If we only bump components, then we have components@2.0 but other packages@1.5. And in a longer time, it could be hard to keep track of versions.